app/models/well.rb

```diff
@@ -82,6 +82,13 @@ def attach(wells)
   scope :include_map, -> { includes(:map) }
   scope :located_at, ->(location) { joins(:map).where(maps: { description: location }) }
   scope :on_plate_purpose, ->(purposes) { joins(:labware).where(labware: { plate_purpose_id: purposes }) }
+  # added version of scope with includes to avoid multiple calls to LabWare in qc report when getting storage location
```

Comment:
*Labwhere
```suggestion
  # added version of scope with includes to avoid multiple calls to LabWhere in qc report when getting storage location
```